package.json

```diff
@@ -180,6 +180,15 @@
                     "type": "boolean",
                     "description": "%AWS.configuration.enableCodeLenses%",
                     "default": true
+                },
+                "aws.togglePrompts": {
+                    "type": "object",
+                    "description": "%AWS.setting.togglePrompts.description%",
+                    "default": {
+                        "warnBeforeRunCommand": true,
+                        "warnBeforeEnablingExcecuteCommand": true,
+                        "warnBeforeDisablingExecuteCommand": true
```

Comment:
if we explicitly name the keys in the default here, that is a maintenance cost: we must always remember to update this. Can we not instead treat a missing key as `true`, in the code logic?